lib/puppet/transport/Schema/philips_hue.rb

```diff
@@ -0,0 +1,17 @@
+require 'puppet/resource_api'
```

Comment:
Looks like the folder name has a capital S, please change this to lowercase

Reply:
Done

Reply:
`lib/puppet/transport/Schema/philips_hue.rb` Schema still has a capital S.